script/build.sh

```diff
@@ -36,36 +36,38 @@
 set -e
 
 echo "-----------------------------"
-echo "--- build.sh | 2021-09-03 ---"
+echo "--- build.sh | 2021-11-11 ---"
 echo "-----------------------------"
 
 #----------------------------------
 # validate parameters
 #----------------------------------
 
 ROOT_DIR=$(pwd)
-BUILD_DIR=$ROOT_DIR/build
-BIN_DIR=$ROOT_DIR/bin
-SRC_DIR=$ROOT_DIR/engine
+echo "ROOT_DIR: ${ROOT_DIR}"
+BUILD_DIR="${ROOT_DIR}/build"
+echo "BUILD_DIR: ${BUILD_DIR}"
+BIN_DIR="${ROOT_DIR}/bin"
+SRC_DIR="${ROOT_DIR}/engine"
```

Comment:
not blocking: the `echo`'s can be done after the definitions as a block. Just makes for easier reasing.

Reply:
Well, I was going to make this change, but then this PR sat here for two weeks. So I decided just to merge it.